src/common/models/Projection.ts

```diff
@@ -342,22 +348,29 @@ export class Projection {
     summaryWithTimeseries: RegionSummaryWithTimeseries,
     futureDaysToInclude: number,
   ) {
```

Comment:
We used to align the actuals timeseries to match the projections timeseries (filling in nulls for dates that didn't exist).  Now, we essentially do it in both directions.  Insert nulls into the projections timeseries to match the actuals timeseries.